### PR TITLE
[2024a] Cherry-pick: RHOAIENG-15393: update piplock renewal gh action

### PR DESCRIPTION
**Pull request by jiridanek**
https://issues.redhat.com/browse/RHOAIENG-15393

## Description

Brings the https://github.com/opendatahub-io/notebooks/commit/ca80fee9e2b6d3e3e0a1b599f7fe782c21a5252d change to 2024a branch.

## How Has This Been Tested?
* https://github.com/jiridanek/notebooks/actions/runs/12398860419

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by jiridanek** [comment]
/override ci/prow/images
not relevant for the PR

**Comment by jiridanek** [comment]
/override ci/prow/images
not relevant for the PR

**Comment by atheo89** [comment]
Thank you Jiri! :slightly_smiling_face: 
/lgtm

File: .github/workflows/piplock-renewal.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+on:  # yamllint disable-line rule:truthy
+  # Triggers the workflow every Monday at 22pm UTC am 0 22 * * 1
+  schedule:
+    - cron: "0 22 * * 1"
+  workflow_dispatch:  # for manual trigger workflow from GH Web UI
+    inputs:
+      branch:
+        description: 'Specify branch'
+        required: false
+        default: 'main'
```
[the remark this answers is not in the document]
Comment (atheo89) [reply .github/workflows/piplock-renewal.yaml:14]: Right, we should update it and probably disable the cron

[the remark this answers is not in the document]
Comment (atheo89) [reply .github/workflows/piplock-renewal.yaml:14]: https://github.com/opendatahub-io/notebooks/pull/820

**Pull request merged**